app/javascript/mastodon/components/avatar_overlay.jsx

```diff
@@ -1,51 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import ImmutablePropTypes from 'react-immutable-proptypes';
-import { autoPlayGif } from '../initial_state';
-import Avatar from './avatar';
-
-export default class AvatarOverlay extends React.PureComponent {
-
-  static propTypes = {
-    account: ImmutablePropTypes.map.isRequired,
-    friend: ImmutablePropTypes.map.isRequired,
-    animate: PropTypes.bool,
```

Comment:
animate argument is unused.
https://github.com/mastodon/mastodon/blob/f05fb51ecb34fd5ef86fad861e16cfc959fe0b27/app/javascript/mastodon/components/status.jsx#L505
https://github.com/mastodon/mastodon/blob/f05fb51ecb34fd5ef86fad861e16cfc959fe0b27/app/javascript/mastodon/features/account_timeline/components/moved_note.jsx#L27
https://github.com/mastodon/mastodon/blob/f05fb51ecb34fd5ef86fad861e16cfc959fe0b27/app/javascript/mastodon/components/__tests__/avatar_overlay-test.jsx#L24